packages/demo/src/components/examples/PartialStringMatchExamples.tsx

```diff
@@ -9,30 +9,30 @@ export class PartialStringMatchExamples extends React.Component<any, any> {
             <div className="mt2">
                 <div className="form-group">
                     <label className="form-control-label">PartialStringMatch without match</label>
-                    <div className="text-dark-grey">
+                    <div>

```

Comment:
PartialStringMatch don't seem to highlight anything anymore. 

### Before:
![image](https://user-images.githubusercontent.com/35579930/106810421-ea4d4180-663a-11eb-846b-24fa5c5578f1.png)
![image](https://user-images.githubusercontent.com/35579930/106810462-fa652100-663a-11eb-8e36-8ece71295546.png)


### After:
![image](https://user-images.githubusercontent.com/35579930/106810268-b40fc200-663a-11eb-8ef1-aee9de452c77.png)
![image](https://user-images.githubusercontent.com/35579930/106810362-d6094480-663a-11eb-9af6-b1980af1e3db.png)